templates/aide.conf.erb

```diff
@@ -12,6 +12,3 @@ report_url=<%= report_url %>
 <%= a %>
 <% end -%>
 
-<% @rules.each do |rule| -%>
```

Comment:
I think we still need this to go along with the legacy rules Array in case people plop files in by hand (yeah, they shouldn't but it's the current behavior) to preserve backward compatibility.

I'll update the PR.

Reply:
If they plop them in by hand, the fix to `purge` will mean puppet deleting them.

Reply:
Ah...good point :-|. OK, never mind, works as originally designed (but was broken) 👍